src/SentiCR/SentiCR.py

```diff
@@ -0,0 +1,367 @@
+from __future__ import print_function
```

Comment:
This directory appears to just be a fork of https://github.com/senticr/SentiCR/tree/master/SentiCR. I'd recommend:
1) Adding the LICENSE file to this directory
2) Add a README to this directory, noting that it's a fork. 
3) (Optional) If you've made modifications to this directory, the most engineering-y way to do this would be to commit the file as-is, then make the changes in a follow up PR.

Reply:
oh.... I'll do 1 and 2 then. I made several changes needed for bazel

Reply:
I like the bazel change. Were you going to add those files in this PR?

Reply:
these files are already after bazel modification... I didn't push the original ones